dockers/docker-fpm-frr/base_image_files/rvtysh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# The command rtysh has root priviledge, only allow to execute readonly commands.
+
+# The options in the show command cannot contains any charactors to run multiple sub-commands potentially, such as "\n", "\r", "|", "&", "$" and ";".
+if printf -- "$*" | grep -qPz '[\n\r|&$;]'; then
+    echo "Not allow to run the command, please use the comand 'sudo vtysh' instead." 1>&2
+    exit 1
+fi
+
+# The sub commands must start with "show"
+LAST_PARA=
+for para in "$@"
+do
+    if [ "$LAST_PARA" == "-c" ] && [[ "$para" != show*  ]]; then
```

Comment:
>"-c" [](http://example.com/codeflow?start=25&length=4)

Could you reinforce only allow one `-c` clause?

Reply:
Maybe we can keep the current behavior, only fix the issue relative to security.